presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/SimpleTtlNodeSelector.java

```diff
@@ -227,8 +227,8 @@ public SplitPlacementResult computeAssignments(Set<Split> splits, List<RemoteTas
     private boolean isTtlEnough(ConfidenceBasedTtlInfo ttlInfo, Duration estimatedExecutionTime)
     {
         Instant expiryTime = ttlInfo.getExpiryInstant();
-        long timeRemaining = MILLIS.between(Instant.now(), expiryTime);
-        return new Duration(Math.max(timeRemaining, 0), TimeUnit.MILLISECONDS).compareTo(estimatedExecutionTime) >= 0;
+        long timeRemaining = SECONDS.between(Instant.now(), expiryTime);
```

Comment:
`timeRemainingInSeconds` will make it clearer

Reply:
Done